lib/hterm.js

```diff
@@ -28,8 +29,8 @@ hterm.Terminal.prototype.overlaySize = function () {};
 // a non-collapsed selection whose text is '', and results
 // in an infinite copy loop
 hterm.Terminal.prototype.copySelectionToClipboard = function () {
-  var text = this.getSelectionText();
-  if (text != null && text !== '') {
+  const text = this.getSelectionText();
+  if (text !== null && text !== '') {
```

Comment:
same here, text as `undefined` would get through `text !== null`  but not `text != null`. Maybe `if (text)` would suffice